sal_interfaces/HVAC/HVAC_Commands.xml

```diff
@@ -1,30 +1,309 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
-<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
-<SALCommand>
-    <Subsystem>HVAC</Subsystem>
-    <Version>3.8</Version>
-    <Author></Author>
-    <EFDB_Topic>HVAC_command_setHvacConfiguration</EFDB_Topic>
-    <Alias>setHvacConfiguration</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-    <item>
-        <EFDB_Name>maxValveFlow</EFDB_Name>
-        <Description>Maximum flow.</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>m3 / s</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>minValveFlow</EFDB_Name>
-        <Description>Minimum flow.</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>m3 / s</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand></SALCommandSet>
\ No newline at end of file
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl'                                              href='http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl'?>
```

Comment:
Why this change? The added spaces before `href` is surprising.

Reply:
This was due to a coding error of mine where I wrapped a long string over two lines and indented it. That has been fixed now.